src/Serializer/RequestsSerializer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Serializer;
+
+use OpenEuropa\EPoetry\PropertyInfo\Extractor\MappingReflectionExtractor;
+use OpenEuropa\EPoetry\Type\CreateRequests;
+use Symfony\Component\Serializer\Encoder\YamlEncoder;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
+use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * Class RequestsSerializer.
+ */
+final class RequestsSerializer implements SerializerInterface
+{
+    /**
+     * The serializer.
+     *
+     * @var \Symfony\Component\Serializer\Serializer
+     */
+    private $serializer;
+
+    /**
+     * RequestsSerializer constructor.
+     */
+    public function __construct()
+    {
+        $this->serializer = new Serializer(
+            [
+                new ArrayDenormalizer(),
+                new ObjectNormalizer(
+                    null,
+                    new CamelCaseToSnakeCaseNameConverter(),
+                    null,
+                    new MappingReflectionExtractor()
+                ),
+            ],
+            [
+                new YamlEncoder(),
```

Comment:
Do we need to set an encoder here? And shouldn't this class extend `Symfony\Component\Serializer\Serializer`?

Reply:
Can we add the encoder XML?

Reply:
Fixed.